draft-ietf-quic-transport.md

```diff
@@ -3804,6 +3804,13 @@ number length, connection ID length, and path MTU.  A receiver MUST accept
 packets containing an outdated frame, such as a MAX_DATA frame carrying a
 smaller maximum data than one found in an older packet.
 
+A sender SHOULD avoid retransmitting information from packets once they are
+acknowledged. This includes packets that are acknowledged after being declared
+lost. Doing so requires senders to retain information about packets after they
+are declared lost. A sender can discard this information after a period of time
+elapses, such as three times the PTO (Section 6.2 of {{QUIC-RECOVERY}}), or on
```

Comment:
```suggestion
are declared lost. A sender can discard this information a period of time after the
packet is lost, such as after the Smoothed RTT (Section 5 of {{QUIC-RECOVERY}}), or on
```

Reply:
I'm happy to change it to 1 x PTO. I'd like the ack delay and the rttdev to be included in this suggestion.

Reply:
ack delay shouldn't need to be included, because the packet would be acknowledged immediately.

Including RTTVar implies that the network's reordering is sometimes as much 3*SRTT, but assuming the data was retransmitted, you'd expect the retransmission to be acknowledged in about 1 RTT, so suggesting you might need to wait longer seems odd to me.